### DISCUSSION TITLE
Custom order / sort of collections

### DISCUSSION THREAD

**Discussion by dragonglobal** [directus/directus#2788] [Feature Requests]
Hello, 

Is it possible to order the collections within your admin panel?

For example, i create 5 collections ( articles, lookbook, pages, settings, blocks )

But in the admin side panel id like them ordered as ( articles, pages, blocks, lookbook, settings )

Is this possible?

**Comment by benhaynes** [comment]
Yeah, in the "Collection Listing" (in Settings):

https://docs.directus.io/guides/roles.html#collection-listing

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
